src/lib/ts/tsconfig.ts

```diff
@@ -32,6 +32,9 @@ export function readDefaultTsConfig(fileName?: string): ng.ParsedConfiguration {
     // these are required to set the appropriate EmitFlags
     flatModuleId: 'AUTOGENERATED',
     flatModuleOutFile: 'AUTOGENERATED',
+
+    // performance
+    skipLibCheck: true, // don't type-check .d.ts files for performance and memory reasons.
```

Comment:
There's also `skipDefaultLibCheck` which would continue to type-check d.ts files, but that one is deprecated in favor of `skipLibCheck` itself. See also microsoft/TypeScript#31417 for a relevant discussion on this flag.

Reply:
I think we should leave this one out.